src/app/page.tsx

```diff
@@ -1,9 +1,81 @@
+'use client';
```

Comment:
sorry, could we actually make the footer into a component? This can allows us to add the footer to other pages if we choose to do so in the future, and it can help the home page look more organized by avoiding a lot of lines of code